docs/imaging_etcher.md

```diff
@@ -0,0 +1,50 @@
+# Imaging with Etcher
+
+## What You'll Need
+  * A computer running Windows, MacOS or Linux
+  * A micro USB cable
+  * Your Amplipi
+
+## 1. Get the Latest AmpliPi Image
+  Download the latest AmpliPi image from [here](https://drive.google.com/file/d/1VHQhHivWCNVwmHukqtjjWu322Nxu_iDp/view) and save it to your computer.
+
+## 2. Get RPIBoot
+Download and install the latest version of RPIBoot from [here](https://github.com/raspberrypi/usbboot/raw/master/win32/rpiboot_setup.exe) on windows, otherwise refer to the instructions for [building RPIBoot](https://github.com/raspberrypi/usbboot#building).
+
+## 3. Get Etcher
+Download and install (if necessary) the latest version of Etcher from [etcher.io](https://etcher.io/), on windows use the portable version.
+
+## 4. Connect your AmpliPi to your computer
```

Comment:
Is there a step here missing, to launch rpiboot? It didn't detect it automagically on my box, but perhaps mine is different.